docs/docs/integrating-with-other-libraries.md

```diff
@@ -0,0 +1,270 @@
+permalink: docs/integrating-with-other-libraries.html
+---
+
+React can be used in any web application. It can be embedded in other applications and, with a little care, other applications can be embedded in React. This guide will examine some of the more common use cases, focusing on integration with jQuery and Backbone.
+
+## Integrating with DOM Manipulation Plugins
+
+React is unaware of changes made to the DOM outside of React. It determines updates based on its own internal representation, and if those updates are invalidated, React has no way to recover.
+
+This does not mean it is impossible or even necessarily difficult to combine React with other ways of affecting the DOM, you just have to be mindful of what each are doing.
+
+The easiest way to avoid conflicts is to prevent the React component from updating. This can be done explicitly by returning false from [`shouldComponentUpdate()`](/react/docs/react-component.html#shouldcomponentupdate), or by rendering elements that have no reason to change.
+
+```js
+class SomePlugin extends React.Component {
+  componentDidMount() {
+    this.$el = $(this.el);
+    this.$el.somePlugin();
+  }
+
+  componentWillUnmount() {
+    this.$el.somePlugin('destroy');
+  }
+
+  render() {
+    return <div ref={el => this.el = el} />
+  }
+}
+```
+A [ref](/react/docs/refs-and-the-dom.html) is used to pass the underlying DOM element to the plugin. The `<div>` element has no properties or children, so React has no reason to update it, leaving the plugin free to update the DOM without conflicts.
+
+The component still has to be unmounted, which provides one final opportunity for conflict. If the plugin does not provide a method for cleanup, you will probably have to provide your own, remembering to remove any event listeners the plugin registered to prevent memory leaks.
+
+To demonstrate these concepts let's write a minimal wrapper for the plugin [Chosen](https://github.com/harvesthq/chosen), which augments `<select>` inputs.
+
+Chosen does not render the initial select, so it is up to React to do so. Chosen hides the initial select and creates its own control, notifying the original of changes using jQuery. Because it is Chosen maintaining the state, it is easiest to implement the wrapper using an [uncontrolled component.](/react/docs/uncontrolled-components.html)
+
+```js
+class Chosen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleUpdate = this.handleUpdate.bind(this);
+  }
+
+  handleUpdate(event) {
+    this.props.update(event.target.value);
+  }
+
+  componentDidMount() {
+    this.$el = $(this.el);
+    this.$el.chosen();
+    this.$el.on('change', this.handleUpdate);
+  }
+
+  componentDidUpdate() {
+    this.$el.trigger('chosen:updated');
+  }
+
+  componentWillUnmount() {
+    this.$el.off('change', this.handleUpdate);
+    this.$el.chosen('destroy');
+  }
+
+  render() {
+    return (
+      <select ref={el => this.el = el}>
+        {this.props.children}
```

Comment:
I'm worried that this example is potentially dangerous because we're mixing React children with jQuery parent with React grandparents. Maybe it would be better to use a "leaf" component for a jQuery example.

It's also worth noting that generally it is recommended to use React components, and this is just an escape hatch for when you need to integrate with older code.